selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -106,12 +112,31 @@ public ProjectExplorer(
     this.editor = editor;
     this.testWebElementRenderChecker = testWebElementRenderChecker;
     this.browserLogsUtil = browserLogsUtil;
-    this.seleniumWebDriverHelper = seleniumWebDriverHelper;
-    loadPageTimeout = new WebDriverWait(seleniumWebDriver, LOAD_PAGE_TIMEOUT_SEC);
-    redrawUiElementsWait = new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
+    this.helper = seleniumWebDriverHelper;
```

Comment:
`helper` is too short name for the field, because it doesn't explain what kind of help it proposes.

Reply:
I agree with you this is not very convenient but this is my point - this object uses almost in each project explorer's method, and that's why I did it shorter, and you just need one time used "F4"  to see what it means and remember it for all cases.

Reply:
I understood why you shorten the name. From the same reason it could be named it just "h" :-)
But it is not good idea to force readers to find the type of field to understand what does it mean.

Reply:
Ok, I will change it

Reply:
But we can get rid of SeleniumWebDriverHelper by moving it's staff to the SeleniumWebDriver class itself.